<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketWidget</class>
 <widget class="QWidget" name="MarketWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MarketWidget{
	
	background-image: url(:/image/climain.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>企业管理平台</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>140</y>
     <width>1125</width>
     <height>730</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_file">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>正在链接文件服务器...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>164</width>
     <height>730</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pb_home">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>商品管理</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pb_shop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>店铺管理</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pb_repertory">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>库存监测</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pb_bill">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>利润统计</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pb_chat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>聊天系统</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pb_user">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>修改密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>当前时间:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lb_time">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
	
	background-image: url(:/ico/common/ic_close_white.png);

background-color: rgb(16, 59, 114);

}

QPushButton:hover
{
	
	background-image: url(:/ico/common/ic_close_white.png);

background-color: rgb(24, 102, 177);

}

QPushButton:pressed{
	
	background-image: url(:/ico/common/ic_close_white.png);

	background-color: rgb(16, 59, 114);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/resource/common/ic_close_white.png</normaloff>:/resource/common/ic_close_white.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_min">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
	
	background-image: url(:/ico/common/ic_min_white.png);
background-color: rgb(16, 59, 114);

}

QPushButton:hover
{
	
	background-image: url(:/ico/common/ic_min_white.png);
background-color: rgb(24, 102, 177);

}

QPushButton:pressed{
	
	background-image: url(:/ico/common/ic_min_white.png);
	background-color: rgb(16, 59, 114);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/resource/common/ic_min.png</normaloff>:/resource/common/ic_min.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>80</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="pb_first">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
      </property>
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pb_quit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
